.github/workflows/performance.yml

```diff
@@ -57,7 +57,7 @@ jobs:
                   ./bin/plugin/cli.js perf "wp/$WP_MAJOR" "$PREVIOUS_RELEASE_BRANCH" "$CURRENT_RELEASE_BRANCH" --tests-branch $GITHUB_SHA --wp-version "$WP_MAJOR"
 
             - name: Compare performance with base branch
-              if: github.event_name == 'push'
+              if: github.event_name == 'pull_request'
```

Comment:
This should be reverted right?

Reply:
Yes, this change was made temporarily to allow pull requests to also run the performance test compared with base branch. Unnecessary changes were reverted in 467f513.

Reply:
Just noting that changing the event name like that in PRs had some unwanted side effects, basically all the temporary commits in this PR started logging numbers in codevitals.run 

I think we should stop doing that in the future. (at least for the part that calls the log script)

Reply:
Do we have some sort of permission check as GH Actions does for `pull_request` and `pull_request_target`? Does this mean that anyone opening a PR could potentially skew the metrics or even possibly run malicious code in codevitals? Or is it scoped only under maintainers in this repo? Asking to be sure this is not a security risk. Please ignore me if this has already been thought of though!

Reply:
> Does this mean that anyone opening a PR could potentially skew the metrics or even possibly run malicious code in codevitals?

Anyone that has Gutenberg permissions, If I'm not wrong, fork PRs don't run CI unless approved.